platforms/jvm/plugins-jvm-test-suite/src/integTest/groovy/org/gradle/api/plugins/JvmTestSuitePluginIntegrationTest.groovy

```diff
@@ -110,76 +110,6 @@ Artifacts
         hasIncubatingLegend()
     }
 
-    def "Only one suite with a given test type allowed per project"() {
```

Comment:
These tests were moved to `TestReportAggregationPluginIntegrationTest.groovy`. See the [commit message](https://github.com/gradle/gradle/pull/29190/commits/b017cfced3cfee701fbf4c93441232851871231e) for more details.